File: src/main/java/com/cartobucket/auth/routes/mappers/UserRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.UserRequestFilter;

import java.util.List;
import java.util.UUID;

public class UserRequestFilterMapper {
    public static UserRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new UserRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/UserRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.UserRequestFilter
import java.util.UUID

object UserRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): UserRequestFilter =
        UserRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
